week-3/slideshow/slideshow.js

```diff
@@ -5,4 +5,40 @@ const images = [
 ];
 
 
-// Write your code here
\ No newline at end of file
+// Write your code here
+
+const forwbutton = document.getElementById("backward-btn");
+const backwbutton = document.getElementById("forward-btn");
+const imgcontent = document.getElementById("carousel-img");
+const autoForward = document.getElementById("auto-forward-btn");
+const autobackward = document.getElementById("auto-backward-btn");
+const autoStop = document.getElementById("stop");
+let autoIntervalId;
+let currentIndex = 0;
+
+function forwarding(){
+    currentIndex = (currentIndex+1 )%images.length;
+    imgcontent.src = images[currentIndex];
+};
+
+function backWarding(){
+    currentIndex = (currentIndex   + 2) % images.length;
+    imgcontent.src = images[currentIndex];
+};
+
+forwbutton.addEventListener("click", forwarding);
+
+backwbutton.addEventListener("click", backWarding);
+
+autoForward.addEventListener("click", ()=> {
+    autoIntervalId = setInterval(forwarding,2000);
+     });
+
+autobackward.addEventListener("click", ()=> {
```

Comment:
good Job with the use of Event listeners and intervals! Consider what would happen if you click the autoforward/autobackwards multiple times